web/validators/support_author_validator.ex

```diff
@@ -0,0 +1,34 @@
+defmodule ParticipateApi.SupportAuthorValidator do
+  import Ecto.Changeset
+  import Ecto.Query
+  alias ParticipateApi.Support
+  alias ParticipateApi.Repo
+
+  def validate_no_previous_support_given_by_author(changeset) do
+    author_id = get_field(changeset, :author_id)
+
+    query = from Support, where: [author_id: ^author_id]
+
+    if Repo.one(query) do
```

Comment:
Good catch. Sloppy code on my part regarding the missing `proposal_id` select :P 

And I was treating `Repo.one` as Rails' `Model.first`, I didn't know it barfed when finding more than one result. Both fixed now